src/batches/setBatchLesionSegmentation.m

```diff
@@ -19,21 +19,21 @@
   [anatImage, anatDataDir] = getAnatFilename(BIDS, subLabel, opt);
   unified_segmentation.step1data{1} = fullfile(anatDataDir, anatImage);
 
-  % define SPM folder
-  spmDir = spm('dir');
-
-  % specify Prior EXTRA class (lesion prior map)
-  lesionPriorMap = fullfile(spmDir, 'toolbox', 'ALI', 'Priors_extraClass', 'wc4prior0.nii');
-
-  unified_segmentation.step1prior = {lesionPriorMap};
-
-  unified_segmentation.step1niti = 2;                     % number of iterations
-  unified_segmentation.step1thr_prob = 0.333333333333333; % threshold probability
-  unified_segmentation.step1thr_size = 0.8;               % threshold size (in cm3)
-  unified_segmentation.step1coregister = 1;               % coregister in MNI space (yes: 1)
-  unified_segmentation.step1mask = {''};                  % specify cost function mask(optional)
-  unified_segmentation.step1vox = 2;                      % Voxel sizes (in mm)
-  unified_segmentation.step1fwhm = [8 8 8];               % Smooth: FWHM
+%   % define SPM folder
```

Comment:
What I think we will do here is to grab the default options you define in `ALI_my_defaults` and "reinject" them in this structure.

Let's chat on Monday so I can brief you on how to do that in details.